push/Push.cpp

```diff
@@ -226,7 +226,7 @@ std::string CPush::ProcessSendValue(const std::string &rawsendValue, const int d
 	}	
 	else if (vType == "Counter")
 	{
-		sprintf(szData,"Not supported yet");
+		sprintf(szData,"%d", atoi(rawsendValue.c_str()));
```

Comment:
simpel, you making a string from it...
and it is already a string
so just a 'strcpy' would work (mind the .c_str(), because we need to get it from a std::string)